packages/scss/src/components/_actionIcon.scss

```diff
@@ -0,0 +1,20 @@
+.actionIcon {
+	background: transparent;
+	border: 0;
+	border-radius: 50%;
+	color: _color("text.default");
+	cursor: pointer;
+	height: 2rem;
```

Comment:
Here is a test with em : 
![image](https://user-images.githubusercontent.com/25581936/27726258-fc519598-5d79-11e7-8501-03cccaed4788.png)
If I reduce it's size, default will be too small. Si how should I deal with sizing, then ? :/

I'm really attached to have encapsuled component. Furthermore, even basic icons have a .size-xxx option. So why especially this component should be parent relative?